scripts/database/pg/migrations/013.do.add-frequency.sql

```diff
@@ -0,0 +1,9 @@
+DO $$
+    BEGIN
+        BEGIN
+            ALTER TABLE cd_dojos ADD COLUMN frequency character varying;
```

Comment:
Could we make this an enum instead please? I think it would be faster than a character. We could add an option of "other" then too.

The SQL would look something like:
```
CREATE TYPE frequency_type AS ENUM ('weekly', 'bi-weekly', 'monthly', 'other');
ALTER TABLE cd_dojos ADD COLUMN frequency frequency_type not null default 'other';
```